<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>clsWKEControlBox</class>
 <widget class="QDialog" name="clsWKEControlBox">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>617</width>
    <height>436</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QDialog
{
	background-color:rgb(150,150,251);
}</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>48</y>
     <width>301</width>
     <height>151</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 12pt &quot;楷体&quot;;
background-color: rgb(255, 170, 0);</string>
   </property>
   <property name="title">
    <string>Channel 1</string>
   </property>
   <layout class="QGridLayout" name="gridLayout_3">
    <item row="0" column="0">
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>LCR condition</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_3">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QRadioButton" name="rbtnLCR1">
        <property name="text">
         <string>L,Q,100kHz,1V,FAST,Auto,Parallal</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="1" column="0">
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Hight voltage condition</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_4">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QRadioButton" name="rbtnHV1">
        <property name="text">
         <string>L,Q,100kHz,1V,FAST,Auto,Parallal</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>306</x>
     <y>48</y>
     <width>290</width>
     <height>127</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 12pt &quot;楷体&quot;;
background-color: rgb(255, 255, 0);</string>
   </property>
   <property name="title">
    <string>Channel 2</string>
   </property>
   <layout class="QGridLayout" name="gridLayout_2">
    <property name="leftMargin">
     <number>6</number>
    </property>
    <property name="topMargin">
     <number>3</number>
    </property>
    <property name="rightMargin">
     <number>6</number>
    </property>
    <property name="bottomMargin">
     <number>6</number>
    </property>
    <item row="0" column="0">
     <widget class="QGroupBox" name="groupBox_6">
      <property name="title">
       <string>LCR condition</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_7">
       <property name="topMargin">
        <number>3</number>
       </property>
       <property name="bottomMargin">
        <number>3</number>
       </property>
       <item row="0" column="0">
        <widget class="clsLabel" name="labelLCR2">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>23</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">font: 12pt &quot;楷体&quot;;
background-color: rgb(0, 255, 255);</string>
         </property>
         <property name="text">
          <string>100KHz,1V,L,Q,Fast,Auto,Parallal</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QGroupBox" name="groupBox_7">
      <property name="title">
       <string>Hight voltage condition</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_6">
       <property name="topMargin">
        <number>3</number>
       </property>
       <property name="bottomMargin">
        <number>3</number>
       </property>
       <item row="0" column="0">
        <widget class="clsLabel" name="labelHV2">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>23</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">font: 12pt &quot;楷体&quot;;
background-color: rgb(85, 255, 127);</string>
         </property>
         <property name="text">
          <string>100KHz,1V,L,Q,Fast,Auto,Parallal</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QGroupBox" name="groupBox_3">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>250</y>
     <width>296</width>
     <height>130</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 12pt &quot;楷体&quot;;
background-color: rgb(255, 255, 0);</string>
   </property>
   <property name="title">
    <string>Channel 3</string>
   </property>
   <layout class="QGridLayout" name="gridLayout_9">
    <property name="topMargin">
     <number>3</number>
    </property>
    <item row="0" column="0">
     <widget class="QGroupBox" name="groupBox_8">
      <property name="title">
       <string>LCR condition</string>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <property name="topMargin">
        <number>3</number>
       </property>
       <property name="bottomMargin">
        <number>3</number>
       </property>
       <item row="0" column="0">
        <widget class="clsLabel" name="labelLCR3">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>23</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">font: 12pt &quot;楷体&quot;;
background-color: rgb(0, 255, 255);</string>
         </property>
         <property name="text">
          <string>100KHz,1V,L,Q,Fast,Auto,Parallal</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QGroupBox" name="groupBox_9">
      <property name="title">
       <string>Hight voltage condition</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_8">
       <property name="topMargin">
        <number>3</number>
       </property>
       <property name="bottomMargin">
        <number>3</number>
       </property>
       <item row="0" column="0">
        <widget class="clsLabel" name="labelHV3">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>23</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">font: 12pt &quot;楷体&quot;;
background-color: rgb(85, 255, 127);</string>
         </property>
         <property name="text">
          <string>100KHz,1V,L,Q,Fast,Auto,Parallal</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>5</y>
     <width>123</width>
     <height>38</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="btnNewSetup">
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>36</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 10pt &quot;楷体&quot;;</string>
      </property>
      <property name="text">
       <string>新建
步骤</string>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
  <widget class="QGroupBox" name="groupBox_10">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>250</y>
     <width>296</width>
     <height>130</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 12pt &quot;楷体&quot;;
background-color: rgb(255, 170, 0);</string>
   </property>
   <property name="title">
    <string>Channel 4</string>
   </property>
   <layout class="QGridLayout" name="gridLayout_10">
    <property name="topMargin">
     <number>3</number>
    </property>
    <item row="0" column="0">
     <widget class="QGroupBox" name="groupBox_11">
      <property name="title">
       <string>LCR condition</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_11">
       <property name="topMargin">
        <number>3</number>
       </property>
       <property name="bottomMargin">
        <number>3</number>
       </property>
       <item row="0" column="0">
        <widget class="clsLabel" name="labelLCR4">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>23</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">font: 12pt &quot;楷体&quot;;
background-color: rgb(0, 255, 255);</string>
         </property>
         <property name="text">
          <string>100KHz,1V,L,Q,Fast,Auto,Parallal</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QGroupBox" name="groupBox_12">
      <property name="title">
       <string>Hight voltage condition</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_12">
       <property name="topMargin">
        <number>3</number>
       </property>
       <property name="bottomMargin">
        <number>3</number>
       </property>
       <item row="0" column="0">
        <widget class="clsLabel" name="labelHV4">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>23</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">font: 12pt &quot;楷体&quot;;
background-color: rgb(85, 255, 127);</string>
         </property>
         <property name="text">
          <string>100KHz,1V,L,Q,Fast,Auto,Parallal</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>390</y>
     <width>204</width>
     <height>35</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <spacer name="horizontalSpacer_2">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QPushButton" name="btnCancel">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>33</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 12pt &quot;楷体&quot;;</string>
      </property>
      <property name="text">
       <string>Cancel</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btnOK">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>33</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 12pt &quot;楷体&quot;;</string>
      </property>
      <property name="text">
       <string>OK</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>clsLabel</class>
   <extends>QLabel</extends>
   <header>clsLabel.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
